examples/tutorials/data/veritas.py

```diff
@@ -0,0 +1,459 @@
+# files are processed with EventDisplay, but VEGAS analysis should be
+# identical apart from the integration region size, which is handled by `RAD_MAX`.
+#
+
+data_store = DataStore.from_dir("$GAMMAPY_DATA/veritas/crab-point-like-ED")
+data_store.info()
+
+
+######################################################################
+# We filter our data by only taking observations within :math:`5 \degree`
+# of the Crab Nebula. Further details on how to filter observations can be
+# found in :doc:`user-guide/dl3.html#getting-started-with-data`.
+#
+
+target_position = SkyCoord(83.6333, 22.0145, unit="deg")
+
+selected_obs_table = data_store.obs_table.select_sky_circle(target_position, 5 * u.deg)
+obs_ids = selected_obs_table["OBS_ID"]
+
+observations = data_store.get_observations(obs_id=obs_ids, required_irf="point-like")
+
+
+######################################################################
+# Peek the first run in the data release
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+
+
+######################################################################
+# Peek at the IRFs included. You should verify that
+# the IRFs are filled correctly and that there are no values set to zero
+# within your analysis range. We can also peek at the effective area
+# (`aeff`) or energy migration matrices (`edisp`) with the `peek()`
+# method.
+#
+
+observations[0].peek()
+
+
+######################################################################
+# Peek the events and their time/energy/spatial distributions.
+#
+
+observations[0].events.peek()
+
+
+######################################################################
+# Part II: Estimate counts and significance
+# -----------------------------------------
+#
+
+
+######################################################################
+# Set the energy binning
+# ~~~~~~~~~~~~~~~~~~~~~~
+#
+# The energy axis will determine the bins in which energy is calculated,
+# while the true energy axis defines the binning of the energy dispersion
+# matrix and the effective area. Generally, the true energy axis should be
+# more finely binned than the energy axis and span a larger range of
+# energies, and the energy axis should be binned to match the needs of
+# spectral reconstruction.
+#
+# Note that if the `~gammapy.makers.SafeMaskMaker` (which we will define
+# later) is set to exclude events below a given percentage of the
+# effective area, it will remove the entire bin containing the energy that
+# corresponds to that percentage. Additionally, spectral bins are
+# determined based on the energy axis and cannot be finer or offset from
+# the energy axis bin edges. See
+# :doc:`/tutorials/api/makers html#safe-data-range-handling`\ for more
+# information on how the safe mask maker works.
+#
+
+energy_axis = MapAxis.from_energy_bounds("0.01 TeV", "100 TeV", nbin=100)
+energy_axis_true = MapAxis.from_energy_bounds(
+    "0.01 TeV", "100 TeV", nbin=200, name="energy_true"
```

Comment:
typically the `energy_axis_true` should span a larger range than the reconstructed axis. This is to correctly take into account the spill over at the lower energies